_posts/2024-11-06-#30239.md

```diff
@@ -10,18 +10,41 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+- Recommended reading:
+  - [#30352: Add PayToAnchor(P2A), OP 1 <0x4e73> as standard output script for spending](https://bitcoincore.reviews/30352)
+
+- Bitcoin Core considers a transaction output to be [dust](https://bitcoin.stackexchange.com/questions/10986/what-is-meant-by-bitcoin-dust) when its value is lower than the cost of creating and spending it.
 
+- PR [30352](https://github.com/bitcoin/bitcoin/pull/30352) added support for PayToAnchor (P2A), a new output type that enables highly efficient keyless anchors (with very low vbyte usage). This new anchor type offers an improvement over previous anchor types (some of which relied on more expensive constructs such as P2WSH).
+
+- The ability of anchors to use amounts below the dust threshold was split off into the ephemeral dust PR we're reviewing today. Ephemeral dust enables additional flexibility and efficiency, for example to enable keyed and keyless anchors to use zero funds from inputs while still enabling a child to bring fees (ala CPFP). This can be useful for L2s (e.g. for the LN).
 
 ## Questions
 
 1. Did you review the PR? [Concept ACK, approach ACK, tested ACK, or NACK](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review)? What was your review approach?
--->
 
+1. Where in code is it determined if an amount is dust?
+
+1. Is dust restricted by consensus? Policy? Both?
+
+1. How can dust be problematic?
+
+1. What is _ephemeral_ dust (why is the term _ephemeral_ significant)? What are the proposed rules specific for ephemeral dust?
+
+1. Why is it important to impose a fee restriction? Which specific vulnerabilities can you imagine without the fee restriction? Can you think of (in your view) positive use cases that are not possible because the fee restriction?
+
+1. How are 1P1C relay and TRUC transactions relevant to ephemeral dust?
+
+1. What could happen if restrictions on the the child tx were not in place?
+
+1. Can a node operator change the amounts considered to be dust? If so, how? How are ephemeral dust rules enforced in this case? What would happen if the node tries to broadcast a transaction after adjusting dust amounts?
+
+1. Could a miner choose to mine transactions containing dust? Is this a concern? What could be done to address the concerns, if any?
+
+### Bonus questions
+1. Which types of tests were created for the PR? What is the purpose of each type of test?
```

Comment:
I think you could lean in to this question and expand it. It'd be great to encourage people to review tests more closely, run the benchmark and share their results, run the fuzzers, etc. I think it's a greater portion of the code than the validation logic. It would also give the people who attended the previous ephemeral anchors/p2a meetings a new angle for review.

Reply:
Thanks. Expanded the question and added another pertaining to coverage. I think good discussion can happen here with the audience.